### PR TITLE
Featured branch

### PR DESCRIPTION
**Pull request by Mikey1-nuel**
###  Portfolio: Setup and Mobile first.

In this portfolio we:

- Added a portfolio design responsive to mobile devices.
- Made variations to the portfolio to fit a medium-screen device.
- Added a README file to enable users to understand my project and access it if need be.

### CODE REVIEW

**Review by joel-kalema — CHANGES_REQUESTED** [review]
Hi @Mikey1-nuel 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 

- [ ] You have submitted a project that has linters that does not pass, make sure you solve them and the ask again for a review


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @joel-kalema in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by iambenkis — CHANGES_REQUESTED** [review]
Hi @Mikey1-nuel  , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 

- [ ] You have submitted a project that has linters that do not pass, make sure you solve them and to ask again for a review , check [this link ](https://github.com/microverseinc/linters-config/tree/master/html-css)for linter installation.


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me `@iambenkis` in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700;900&display=swap" rel="stylesheet">
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+</head>
+<body>
+	<section class="header">
```
Comment (Kossi-stack) [inline index.html]: - [ ] The header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+		<nav class="menu">
+			<h4>John Doe</h4>
```
Comment (Kossi-stack) [inline index.html]: - [ ] Kindly use a tag for the logo

File: index.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+			<i class="fa fa-bars"></i>
```
Comment (Kossi-stack) [inline index.html:26]: - [ ] Kindly wrap the hamburger with a nav tag.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+		</nav>
+	</section>
+
+	<section class="bio">
+		<div class="text-area">
+			<h1>Hey There.</h1>
+			<h1>I'm Alex</h1>
```
Comment (Kossi-stack) [inline index.html]: - [ ] The subtitle uses a header tag `hx (examples: <h2>,<h3>, etc...)` you can use h1 only once.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+			<p>I can help you build a product, feature or 
+				website. Look through some of my work and experience!
+				If you like what you see and have a project you need coded,
+				don't hestiate to contact me.</p>
+	        <div class="contact">
+				<i class="fa fa-github"></i>
+				<i class="fa fa-linkedin"></i>
+				<i><img src="images/Vector.png" alt=""></i>
+				<i class="fa fa-twitter"></i>
+				<i class="fa fa-g-mail"><h3>M</h3></i>
```
Comment (Kossi-stack) [inline index.html]: - [ ] Kindly make sure the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @Mikey1-nuel,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 - [ ] Kindly edit the PR's title to something more descriptive. it can be "Portfolio: Setup PR".
 - [ ] Kindly add a descriptive pull request message, This helps other programmers who want to take a look at your code to understand what changes you made to a particular branch, you can find an example [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects).

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700;900&display=swap" rel="stylesheet">
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+</head>
+<body>
+	<header class="header">
+		<div class="container">
+		<div class="logo">
+			<h4>John Doe</h4>
```
Comment (franclobo) [inline index.html]: - [ ] As requested by the previous code reviewer the logo is a link so please 🙏🏽 use the `a` tag for better practices of HTML and CSS.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+			<h3>I am a Software Developer</h3>
+			<p>I can help you build a product, feature or 
+				website. Look through some of my work and experience!
+				If you like what you see and have a project you need coded,
+				don't hestiate to contact me.</p>
+	        <div class="contact">
+				<ul class="social">
+				<li><a href=""><i class="fa fa-github"></i></a></li>
+				<li><a href=""><i class="fa fa-linkedin"></i></a></li>
+				<li><a href=""><i><img src="images/Vector.png" alt=""></i></a></li>
```
Comment (franclobo) [inline index.html]: - **[OPTIONAL]** Kindly add a description for the image in the `alt` attribute for better practices of HTML & CSS.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+				<li><a href=""><i class="fa fa-twitter"></i></a></li>
+				<li><a href=""><i class="fa fa-g-mail"><h3>M</h3></i></a></li>
+			</ul>
+			</div>
+		</div>
+		<img src="images/Illustration.png" alt="" class="png">
```
Comment (franclobo) [inline index.html]: - **[OPTIONAL]** Kindly add a description for the image in the `alt` attribute for better practices of HTML & CSS.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Mikey1-nuel, 

# ♻️ Required Changes ♻️

so-close-jim-halpert

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Correct use of tags 💯 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @Mikey1-nuel 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git Hub-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**